tests/benchmark.py

```diff
@@ -71,9 +71,12 @@ def run_all_tests(folder=None, verbose=True):
         runner = unittest.TextTestRunner()
         for tsi, ts in enumerate(suites):
             for k in ts:
-                for t in k:
-                    print(t.__class__.__name__)
-                    break            
+                try:
+                    for t in k:
```

Comment:
Why is there a for loop if there is an immediate break?

Reply:
When a unit test cannot be imported, the error message was not informative enough, so I caught the exception and raised a new one with more information.